docs/Correlating_Genes.md

```diff
@@ -1,7 +1,7 @@
 <a id="correlating_genes"> </a>
 
 
-Find genes correlating with your gene of interest
+Find genes correlating with your gene of interest test
```

Comment:
klopt deze? of was dit een git test?

Reply:
Ik had even wat problemen met  commit van een subdi 'test'heb ik net weggehaald , vergeten